src/app/app.component.ts

```diff
@@ -7,15 +7,17 @@ import { SkyModalDemoComponent, SkyModalDemoContext } from './skymodaldemo.compo
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'skyuxdialogdemo';
 
   constructor(private skyModalService: SkyModalService) {
-    
+
   }
 
   ngOnInit() {
-    this.openModal('fullScreenModal')
+    setTimeout(() => {
```

Comment:
I discovered a bug with `@skyux/modals`, where the modal will not launch in Angular 7+ inside a `ngOnInit`. You can use a `setTimeout` as a temporary workaround.

Reply:
Issue created: https://github.com/blackbaud/skyux-modals/issues/30